firefox_private_network_privacy_notice/en-US.md

```diff
@@ -0,0 +1,42 @@
+## <span class="privacy-header-firefox">Firefox Private Network (Beta)</span> <span class="privacy-header-policy">Privacy Notice</span>
+Version 2.0, dated December 3, 2019
+{: datetime="2020-12-03" }
+
+## At Mozilla, we design products with your privacy in mind.
+
+This privacy notice explains what data is needed to offer Firefox Private Network (“Service”) and why.
+
+To learn more about how we receive and handle information, and how you can engage with your data, read the [Mozilla Privacy Policy](https://www.mozilla.org/en-US/privacy/).
+
+## Things you should know:
+
+### Firefox Private Network Requires a Firefox Account. 
+This service requires a Firefox Account, which sends Mozilla your email address, locale, and IP address. Learn more about [Firefox Account data practices](https://www.mozilla.org/en-US/privacy/firefox/#firefox-accounts-join-firefox).
+
+### There are two options for Firefox Private Network. 
+1. FPN Browser Protection, which protects your web browsing within Firefox, and
```

Comment:
In the first paragraph, Firefox Private Network (Service), but no FPN mentioned. Should we mention this early so user knows what FPN stands for? I know it is obvious to us, but all of a sudden, it is shows up and it is everywhere.